src/tools/driver/cmd_line_utils.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2014, the Fletch project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE.md file.
+
+library driver.cmd_line_utils;
+
+int getPortNumberFromArg(String arg, String helpText) {
+  final int portNumber = int.parse(arg, onError: (source) {
+    print ("Required argument <port number> was not a number.\n");
```

Comment:
No space after print (generally). Print automatically adds a newline, so are you sure you want an extra one here (and in other places)?